packages/clay-alert/src/ClayAlert.tsx

```diff
@@ -0,0 +1,121 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as React from 'react';
+import classNames from 'classnames';
+import Icon from '@clayui/icon';
+
+type DisplayType = 'danger' | 'info' | 'success' | 'warning';
+
+interface ClayAlertProps extends React.HTMLAttributes<HTMLDivElement> {
+	/**
+	 * Callback function for when the 'x' is clicked.
+	 */
+	onClose?: () => void;
+
+	/**
+	 * Determines the style of the alert.
+	 */
+	displayType?: DisplayType;
+
+	/**
+	 * Flag to indicate the stripe variant.
+	 */
+	stripe?: boolean;
```

Comment:
I'm wondering if we'd better keep the API with that name or `type` (`variant`) and set the `typo` to stripe (`type="stripe"`), it seems more obvious that the stripe is a variation of just defining stripe in the component, I would probably fetch it by variant when I was using this component, even if only have one variant